src/Symfony/Component/Routing/Tests/Fixtures/dumper/url_matcher2.php

```diff
@@ -35,7 +37,7 @@ public function match($pathinfo)
         if (0 === strpos($pathinfo, '/bar')) {
             // bar
             if (preg_match('#^/bar/(?P<foo>[^/]++)$#s', $pathinfo, $matches)) {
-                if (!in_array($this->context->getMethod(), array('GET', 'HEAD'))) {
+                if (!in_array($requestMethod, array('GET', 'HEAD'))) {
```

Comment:
Reading this line makes me think that we could even have `$requestMethod` set to `GET` when it's `HEAD`, so that this condition can be simplified to just `if ('GET' !== $requestMethod) {` WDTY?